assets/js/components.js

```diff
@@ -0,0 +1,41 @@
+export const Navbar = () => {
+  return `
+  <header class="position-fixed top-0 start-0 w-100" style="z-index:999">
+    <nav class="z-index-master navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand fs-1 fw-bold mx-4" href="#">CampBuddy</a>
+      <div class="input-group">
+    </nav>
+    <div class="mt-4 mx-4">
+      <div class="d-flex gap-1">
+        <div class="form-control">
+          <img src="/assets/img/icons/icon-search.png">
+          <input class="search--input" placeholder="Search here..."/>
+        </div>
+        <button class="btn btn-light filter--btn"><img src="/assets/img/icons/icon-filter.png"></button>
```

Comment:
Is this class needed `filter--btn`?

Reply:
Just added for custom styling which I haven't implemented for now... Will remove or keep it in future